docs/components/plugins/AlgoliaSearch.js

```diff
@@ -181,6 +176,11 @@ class AlgoliaSearch extends React.Component {
         },
       ],
     });
+
+    //auto-focuses on mobile
+    if (!this.props.hiddenOnMobile) {
+      document.getElementById('algolia-search-box-mobile').focus();
```

Comment:
A 'more react' way to do this is to use react's `createRef` hook, put a `ref` on the `<input />` element, then call `ref.current.focus()`. The main benefit of using react to call `.focus()` on a `ref` would be if there's ever a case where there are two things with the same id on the same page. I don't think that's a danger here, but something to consider to make this code more maintainable in the future. --im also happy to explain how to do this next week!